src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -633,7 +631,7 @@ def __call__(self):
             if funcName == "repack":
                 self.handleRepackSettings()
 
-            if funcName in ["repack", "merge", "alcaHarvesting" ]:
+            if funcName in ["merge", "alcaHarvesting" ]:
```

Comment:
Periodic remove expression in the pilots has a ceiling of 20GB per core and nothing in the jobs classads can change that.